### DISCUSSION TITLE
Background Update

### DISCUSSION THREAD

**Discussion by ghost** [aguilaair/updat#25] [Q&A] [answered]
Hi there !

Is it possible with this package to add an automatic background update ? I mean the application don't ask the user if he want to update and the app download and install automatically the update in background (for information, I use msix format).

Thanks in advance
Respectfully

**Comment by aguilaair** [comment, marked as answer]
Hey! You can launch the update once that app is closed, so yes! You'll want a slightly modified version of this: https://github.com/aguilaair/updat/blob/main/lib/theme/chips/default_with_silent_download.dart where you never show anything. 

Regarding wether MSIX supports silent installs, you'll have to check with Google unfortunately as I do not know :)
